onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperator.cpp

```diff
@@ -862,7 +862,11 @@ namespace Dml
         {
             return;
         }
-        
+        else if (constExpTensor->IsDataInterface())
```

Comment:
This would work with the current EP implementation, but !IsCpuData would be more precise